server/events/instrumented_project_command_runner.go

```diff
@@ -35,30 +35,31 @@ func NewInstrumentedProjectCommandRunner(scope tally.Scope, projectCommandRunner
 }
 
 func (p *InstrumentedProjectCommandRunner) Plan(ctx command.ProjectContext) command.ProjectResult {
-	return RunAndEmitStats("plan", ctx, p.projectCommandRunner.Plan, p.scope)
+	return RunAndEmitStats(ctx, p.projectCommandRunner.Plan, p.scope)
 }
 
 func (p *InstrumentedProjectCommandRunner) PolicyCheck(ctx command.ProjectContext) command.ProjectResult {
-	return RunAndEmitStats("policy check", ctx, p.projectCommandRunner.PolicyCheck, p.scope)
+	return RunAndEmitStats(ctx, p.projectCommandRunner.PolicyCheck, p.scope)
```

Comment:
Please fill out how this change has been tested in the "tests" section. 

Some devs will run Atlantis locally to confirm and some devs will deploy the code change in their setup to test. Screenshots of the results are helpful here too

Reply:
Working on it

Reply:
Test added

Reply:
the screenshots from the debug and the metric list in the dropdown were generated from my local testing from my branch `albertorm95:bugfix-add-missing-command-name-in-metrics`

Reply:
I confirm I see the stats emitted correctly, and from my testing everything worked end to end as expected, I was able to see the plan, apply, approve_policies, state rm and also see each metric in /metrics endpoint